CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 - Added support for `if_not_exists` parameter during udf and sproc registration.
 - Added `DataFrame.cache_result` and `Series.cache_result` methods for users to persist DataFrames' and Series' to a temporary table lasting the duration of the session to improve latency of subsequent operations.
+- Added an Index class in Snowpark pandas. This class contains a native pandas index for now, but we plan to turn this into a lazily evaluated index in future updates
```

Comment:
Once we finalize whether this applies to just rows or columns as well can you clarify that in the CHANGELOG?

Reply:
yep will do